### PR TITLE
tests: add scenario to assure custom APT configuration is respected

### PR DESCRIPTION
**Pull request by renanrodrigo**
## Why is this needed?
This PR solves all of our problems because it ensures custom APT configuration is respected by the pro client when done globally, whether through proper system configuration or using environment variables.

## Test Steps
CI should be green, as the new integration test runs on X/N lxd-container

## Checklist
 - [ ] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly - that is the whole point
 - [x] Changes here need to be documented, and this was done in: #3151

## Does this PR require extra reviews?
 - [ ] Yes
 - [x] No

### CODE REVIEW

**Comment by renanrodrigo** [comment]
I chose Xenial and Noble because
- Xenial older, Noble newest
- Xenial to Jammy will have APT behaving exactly the same
- Noble carries the new deb822 config files, and we make sure it still works even there

File: features/security_status.feature [hunk 1]
```diff
@@ -859,3 +859,105 @@ Feature: Security status command behavior
...
       """
+
+  Scenario Outline: Pass custom APT configuration to the Client for updates information
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    # Get the system up to date
+    When I apt update
+    And I apt upgrade
+    # Install older versions of packages which have alternatives in -updates and -security
+    # This will mess up a little with the system but all should be fine for the test purpose
+    And I apt install `<packages_to_install>`
```
Comment (orndorffgrant) [inline features/security_status.feature:870]: Nit: please separate this into two variables for the updates update and the security update.

**Review by orndorffgrant — APPROVED** [review]
LGTM

File: features/security_status.feature (hunk 1 above)
Comment (renanrodrigo) [reply features/security_status.feature]: done

**Pull request merged**